### PR TITLE
PWGGA/GammaConv - transition from TParticle to AliVParticle, fix for embedding MCs

### PR DESCRIPTION
**Pull request by dmuhlhei**

### CODE REVIEW

**Comment by dmuhlhei** [comment]
Is there any missing information for approval?

File: PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx [hunk 1]
```diff
@@ -8077,20 +8077,23 @@ Int_t AliAnalysisTaskGammaCalo::CountPhotonsInCluster(Int_t cluslabel)
 }
 void AliAnalysisTaskGammaCalo::DoClusterMergingStudies(AliVCluster* clus,vector<clusterLabel> &labelvect)
 {
+<<<<<<< HEAD
```
Comment (FriederikeBock) [inline PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx:8080]: I doubt this should be here, same as the one below ;)

**Comment by FriederikeBock** [comment]
Hi @dmuhlhei,
thanks for these changes and the work behind it. Did you test all the tasks locally whether they still work as intended? Or at least our major ones?
Regards,
Fredi

**Comment by ktf** [comment]
This PR is from 4 months ago and needs to be rebased. Closing. Feel free to reopen it with a version which does not have conflicts.

**Pull request closed without merging**